src/test/java/baseClasses/FBPageTest.java

```diff
@@ -23,6 +26,15 @@ public void demoTest()
 		driver.close();

```

Comment:
remove this driver.close() as we are closing it in the login() method

Reply:
Sure, I'll do that and revert back asap.

Reply:
Changes Done.